core/controllers/base.py

```diff
@@ -334,18 +334,21 @@ def vaidate_and_normalize_args(self):
         if handler_class_name in handler_class_names_with_no_schema:
             return
 
-        if request_method in ['GET', 'DELETE']:
-            handler_args = {}
-            for arg in self.request.arguments():
+        handler_args = {}
+        payload_arg_keys = []
+        request_arg_keys = []
+        skip_validation_for_args = ['csrf_token']
```

Comment:
ARGS_WHICH_DO_NOT_NEED_VALIDATION

and make it a class or module level constant

Reply:
Done